content/en/docs/onboarding/_index.md

```diff
@@ -0,0 +1,15 @@
+---
+title: "E-Learning for Contributors"
+type: docs
+weight: 9
+description: |
```

Comment:
```suggestion
weight: 2
aliases: [ "/course" ]
description: |
```
Bumps it up in the list and gives it an alias so we can link it like k8s.dev/course